src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Library Controls
     QMenu* libraryMenu = addSubmenu(tr("Library"));
-    addPrefixedControl("[Playlist]", "ToggleSelectedSidebarItem",
-                       tr("Expand/Collapse View"),
-                       tr("Expand/collapse the selected view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveUp",
+                       tr("Move up"),
+                       tr("Equivalent to pressing the UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-
-    addPrefixedControl("[Playlist]", "SelectPlaylist",
-                       tr("Switch Next/Previous View"),
-                       tr("Switch to the next or previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveDown",
+                       tr("Move down"),
+                       tr("Equivalent to pressing the DOWN key on the keyboard"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "MoveVertical",
+                       tr("Move up/down"),
+                       tr("Move vertically in either direction using a knob, as if pressing UP/DOWN keys"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
```

Comment:
> Yes that bad, Interesting that I have not noticed it :-/

So you agree that my proposed new mapping for RMX is in fact more convenient than the old mapping (mouse/keyboard actions no longer required for navigation), and we can safely remove the old deprecated controls from the MIDI learn wizard in order to discourage their use? Mandatory mouse/keyboard action with the current control set was in fact the main motivation for this PR.

> The idea is to add a new key, or modified key to the tree that is "open child" when there is a child and "Tab" when there is no child. Do you think that will work?

I prefer to discuss this outside the context of a merge decision for this PR, as I want to work on the new library layout ASAP, and we seem to agree that the new proposed mapping is better in the current library layout, and MUCH better in the new library layout. OK?

In saying that (i.e. even with the understanding that if we were to add a new control it would not be inside this PR):

In the new library design, "show children" is not required for the most common actions due to the new "button bar" feature selection pane. The only feature which I'm aware of it being required for is "Browse" for navigating the filesystem, so let's assume that scenario.

In this scenario, the most common actions will be change pane and move selection. Show children will be a very uncommon action, so I feel that it's more suitable to map it to the shift key. In this case I don't see any real advantage to having the combined button, since one simply releases the shift key when they're ready to change the pane.

I strongly prefer to keep all navigation controls atomic, so that the code is simple and scalable to different designs. Changing the panes and showing/hiding children in a tree are two completely different things. If you still really want this extra control then please add it yourself in a separate PR?